streamrip/media/album.py

```diff
@@ -106,4 +106,5 @@ def _album_folder(self, parent: str, meta: AlbumMetadata) -> str:
             meta.format_folder_path(formatter), config.filepaths.restrict_characters
         )
 
-        return os.path.join(parent, folder)
+        # limit to 255 characters (max filename length)
+        return os.path.join(parent, folder[:254])
```

Comment:
Why did you limit it to 254 chars? Also, wouldn't the path length include the parent directory?

Reply:
> Also, wouldn't the path length include the parent directory

There's separate limits for filepath and filename lengths. `pathvalidate` hardcodes these values to `4096` and `255` bytes, but they depend on the filesystem used. There's a open issue regarding this though: https://github.com/thombashi/pathvalidate/issues/50